RealmBrowser/Classes/RLMApplicationDelegate.m

```diff
@@ -94,6 +96,27 @@ - (BOOL)application:(NSApplication *)application openFile:(NSString *)filename
     return YES;
 }
 
+- (void)application:(NSApplication *)sender openFiles:(NSArray *)filenames
+{
+    if (filenames.count > kMaxNumberOfFilesAtOnce) {
+        NSString *message = [NSString stringWithFormat:@"Are you sure you wish to open all %lu Realm files?", (unsigned long)filenames.count];
+        
+        NSAlert *alert = [[NSAlert alloc] init];
+        [alert setMessageText:message];
+        [alert setInformativeText:@"Opening too many files at once may result in system instability."];
```

Comment:
I've since changed the wording to "Realm Browser becoming unstable" which should isolate the implications of opening up hundreds of Realm files at once, but still make the user aware. :)